content/recipes/interoperability/bioactivity-profile.md

```diff
@@ -0,0 +1,221 @@
+## Bioactivity data profile
+
+[TOC]
+
+
+
+## Main Objective
+
+To provide guidance on the minimum set of metadata and semantics required to describe any chemical biology experiments, and how to translate bioactivity data to CHEMBL standards. This recipe also applies to binding, functional and ADME or toxicological assay types.
+
+
+## Graphical overview of the Recipe FAIRification Objectives
+
+```mermaid
+graph TD
+    A[Primary data at one concentration]-->B{Number of readouts per molecules?}
+    B-->|one|V{Which unit?}
+    V-->|percentage|D
+    V-->|value|F
+    B-->|multi|C[valid ones]
+    C-->|experiments|D{if triplicates}
+    D-->|aggregating or filtering|E[validated ones]
+    E--> |calculating|G[One average readout value <br> per concentration]
+    G-->F[CC/AC/IC50/IC90 curves]
+```
+
+### BioActivity data
+
+In the chemical biology domain (CBD), bioactivity data refers to datasets coming from high- to low-throughput biochemical and cellular screenings, hit confirmatory assay and profiling assays. Such datasets usually contain IC/AC/CC/EC50 (where 50 can be any number referring to the percent effect of the readout results). 
```

Comment:
```suggestion
In the chemical biology domain (CBD), bioactivity data refers to datasets coming from high- to low-throughput biochemical and cellular screenings, hit confirmatory assay and profiling assays. Such datasets usually contain IC/AC/CC/EC50 (where 50 can be any number referring to the percent effect of the readout results, for instance EC50 stands for `Half maximal Effective Concentration, IC50 stands for `Half maximal Inhibitory Concentration ,TODO:provide all the other definitions). 
```
Any markup from CHEMINF? this is somehow covered further down but I was wondering if this could be brought up there too.